commit a052d62f78200b2cc554d4db6d47f4808d15b113
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Sun Jan 3 21:03:13 2016 +0800

    scsi_scan: don't dump trace when scsi_prep_async_scan() is called twice
    
    (commit a4cf30e15ccea82a07fa5870e8af787b9ea1a672 upstream.)
    
    BugLink: http://bugs.launchpad.net/bugs/1517942
    
    The only user of scsi_prep_async_scan() is scsi_scan_host() and it
    handles the situation correctly. Move 'called twice' reporting to debug
    level as well.
    
    The issue is observed on Hyper-V: on any device add/remove event storvsc
    driver calls scsi_scan_host() and in case previous scan is still running
    we get the message and stack dump on console.
    
    Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
    Reviewed-by: K. Y. Srinivasan <kys@microsoft.com>
    Tested-by: Alex Ng <alexng@microsoft.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    (cherry picked from commit a4cf30e15ccea82a07fa5870e8af787b9ea1a672)
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    Acked-by: Brad Figg <brad.figg@canonical.com>
    Signed-off-by: Kamal Mostafa <kamal@canonical.com>
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/drivers/scsi/scsi_scan.c b/drivers/scsi/scsi_scan.c
index f9f3f82..01ad016 100644
--- a/drivers/scsi/scsi_scan.c
+++ b/drivers/scsi/scsi_scan.c
@@ -1712,8 +1712,7 @@ static struct async_scan_data *scsi_prep_async_scan(struct Scsi_Host *shost)
 		return NULL;
 
 	if (shost->async_scan) {
-		shost_printk(KERN_INFO, shost, "%s called twice\n", __func__);
-		dump_stack();
+		shost_printk(KERN_DEBUG, shost, "%s called twice\n", __func__);
 		return NULL;
 	}
 
